lms/djangoapps/course_api/blocks/api.py

```diff
@@ -17,7 +17,9 @@ def get_blocks(
         requested_fields=None,
         block_counts=None,
         student_view_data=None,
-        return_type='dict'
+        return_type='dict',
+        block_types=None,
+        lti_url=None
```

Comment:
@robrap @lenacom I am also surprised by the additional `ti_url` parameter.  I thought our earlier discussion indicated that we would simply add `lti` as an additional value to `requested_fields`.  So the request would be (copied from above):
```
/api/courses/v1/blocks/<usage_id>/?username=anjali&requested_fields=lti
```